setup.cfg

```diff
@@ -41,17 +41,17 @@ package_dir =
 include_package_data = True
 python_requires = >= 3.8
 install_requires =
-    attrs~=20.3
+    attrs
     black>=19.3b0
-    cached-property~=1.5
-    click~=7.1
+    cached-property
+    click
     dace@git+https://github.com/spcl/dace.git
-    jinja2~=2.10
-    numpy~=1.15
+    jinja2
+    numpy
     packaging>=20.0
-    pybind11~=2.5
-    tabulate~=0.8
-    typing-extensions~=3.7
+    pybind11
+    tabulate
+    typing-extensions
     # ---- eve / gtc ----
     boltons>=20.0
```

Comment:
I assume some or maybe even most of these new dependencies can have these `>=` constraints essentially removed.